compiler/haskell-ide-core/src/Development/IDE/Orphans.hs

```diff
@@ -0,0 +1,51 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE FlexibleInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+-- | GHC utility functions. Importantly, code using our GHC should never:
+--
+-- * Call runGhc, use runGhcFast instead. It's faster and doesn't require config we don't have.
+--
+-- * Call setSessionDynFlags, use modifyDynFlags instead. It's faster and avoids loading packages.
```

Comment:
This comment doesn't make too much sense at this location.

Reply:
Yep, should have stayed with UtilGHC. I'll move it back.